### ISSUE TITLE
[FeatureRequest] Better handling of filling and draining of Fluids Tanks

### ISSUE THREAD

**Issue by IcedReaper** [InnovativeOnlineIndustries/Titanium#76]
**Is your feature request related to a problem? Please describe.**
When you have a machine with more than one fluid tank, the handling of filling and draining is quite clunky.

**Describe the solution you'd like**
I would like to disable the Fill/Drain-GUI-Feature at the right hand side of the machines GUI and would like to fill and drain the tanks directly by draining/filling a tank/bucket on the Tank GUI Element it self.

**Additional context**
Here is a screenshot of the Machine I've done: https://imgur.com/w1dZ9Lw
If I set the Inputs' action to BOTH (to be able to rescue fluids that weren't processed) I always drain the first input first instead of the fluid I want, or at least the first output.
Here you can see that the first Input is drained instead of the Output, or anything else you'd like: https://imgur.com/XyRxeM1

Thanks in advance,
IcedReaper

**Comment by IcedReaper** [comment]
I came up with a solution this morning, but it doesn't feel like it's the correct way.
I added a button per Tank, at the same exact position, and handled the filling and draining over these buttons.
It works, it's quite some code, but like I wrote, I don't think it's the way it's supposed to be.

Here is the pasteBin to the Buttons code and some explanation of the encapsulated code:
https://pastebin.com/HdqDvB8y

I think this should a behavior that should be activateable.
